index.html

```diff
@@ -75,6 +75,225 @@ <h1 id="greeting">I'm Conor Glad to See you!</h1>
+
+        <div class="small_buttons">
+            <ul class="button_list">
+                <li class="language_descriptor">html</li>
+                <li class="language_descriptor">css</li>
+                <li class="language_descriptor">javaScript</li>
+            </ul>
+        </div>
+
+        <button class="see_project" type="button">See Project</button>
+
+
+          
+        </div>
+
+        <div class="project_1">
+
+            <div class="project1_img">
+                <img src="img/tonic2.png" alt="tonic">
+            </div>
+
+        <h3 class="project1_title">Tonic</h3>
+
+        <ul class="tonic_list">
+            <li class="canopy">CANOPY</li>
+            <li> <img class="counter" src="img/Counter.png" alt="tonic"> </li>
+            <li> Back End Dev </li>
+            <li> <img class="counter" src="img/Counter.png" alt="tonic"> </li>
+            <li> 2015 </li>
+        </ul>
+
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+        <div class="small_buttons">
+            <ul class="button_list">
+                <li class="language_descriptor">html</li>
+                <li class="language_descriptor">css</li>
+                <li class="language_descriptor">javaScript</li>
+            </ul>
+        </div>
+
+        <button class="see_project" type="button">See Project</button>
+
+
+          
+        </div>
+
+        <div class="project_1">
+
+            <div class="project1_img">
+                <img src="img/multipoststories2.png" alt="multi-post-stories">
+            </div>
+
+        <h3 class="project1_title">Multi-Post Stories</h3>
+
+        <ul class="tonic_list">
+            <li class="canopy">CANOPY</li>
+            <li> <img class="counter" src="img/Counter.png" alt="counter"> </li>
+            <li> Back End Dev </li>
+            <li> <img class="counter" src="img/Counter.png" alt="counter"> </li>
+            <li> 2015 </li>
+        </ul>
+
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+        <div class="small_buttons">
+            <ul class="button_list">
+            <li class="language_descriptor">html</li>
+            <li class="language_descriptor">css</li>
+            <li class="language_descriptor">javaScript</li>
+            </ul>
+        </div>
+
+        <button class="see_project" type="button">See Project</button>
+
+
+          
+        </div>
+
+        
+    </section> 
+
+    <section id="contact_info">
+        <div>
+            <h1 id="greeting_2">About Myself</h1>
+        </div>
+
+        <div>
+            <p class="about_me">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </div>
+
+        <div>
```

Comment:
- [ ] I kindly suggest that you make use of a header tag from `h2` to `h6` for your `LET'S CONNECT` text because it's a header.